### ISSUE TITLE
Icon Find: Firefox Clipboard API support

### ISSUE THREAD

**Issue by francisrupert** [USDA-FSA/fsa-design-system#459]
For Icon Find development (#413), the Clipboard API was used. However it's not supported by Firefox. 

```
Uncaught ReferenceError: ClipboardItem is not defined
    copyToClipboard http://localhost:4000/fsa-design-system/js/demo/ds-icon-find.js:191
    onclick http://localhost:4000/fsa-design-system/components/icons/:1
```

Elsewhere in the site, the Copy Code function uses [clipboard.js](https://clipboardjs.com/), e.g. https://github.com/USDA-FSA/fsa-design-system/blob/gh-pages/js/components/ds-clipboard.js.

**Comment by bignamehere** [comment]
@francisrupert Interesting... must have missed that error on Firefox. I am trying to remain as vanilla as possible, so using clipboard.js adds a dependency. jQuery is another library that we should try to avoid as well, which seems to be used in ds-clipboard.js.

The Clipboard API is supported by all modern browsers (IE is not a modern browser), but I didn't realize `write()` and `ClipboardItem()` were not. It looks like I don't actually need the `ClipboardItem()` if I just switch to `navigator.clipboard.writeText()`. I recall seeing an example using `ClipboardItem()` to remove the NodeList Object reference and just use the text, but it seems `writeText()` does this for me.

**Comment by bignamehere** [comment]
merged into 1.18.0 release
